build-config/recovery/xorriso-options-arm64-uefi.cfg

```diff
@@ -0,0 +1,31 @@
+# This file is derived from https://github.com/opencomputeproject/onie/blob/master/build-config/recovery/xorriso-options-uefi.cfg
```

Comment:
fixed. Now the bios cfg and uefi cfg are seprated and will be applied based on firmware_type bios or uefi, these both will be merged and applied if firmware_type is selected as auto.